src/components/ui/deployments/DeploymentLayout.js

```diff
@@ -9,4 +10,8 @@ exports.closeEventsPanel = function() {
 
 exports.getNodePropertiesPanel = function(){
     return $('[floating-deployment-node-panel]');
-};
\ No newline at end of file
+};
+
+exports.deleteDeployment = function() {
+        new common.ActionsDropdown($('body')).clickMenuOption('Delete');
```

Comment:
Lets make 'body' the default then.. if no args passed, ActionsDropdown will use 'body' as default.